src/Generation/Generator/Renderer/Internal/ReturnTypeToNativeExpression/Converter/String.cs

```diff
@@ -5,10 +5,16 @@ namespace Generator.Renderer.Internal.ReturnTypeToNativeExpressions;
 internal class String : ReturnTypeConverter
 {
     public bool Supports(AnyType type)
-        => type.Is<GirModel.String>();
+        => type.Is<GirModel.Utf8String>();
```

Comment:
There weren't any cases of filename strings being returned from a callback so I don't think it needs to be implemented yet. This should be fine to rename to Utf8String.cs
If we do need to add support for filename strings, it would be in a separate file since different code is needed to generate a native filename string and transfer ownership to the native caller

Reply:
Yes we can add it later :+1: